aliases/available/uuidgen.aliases.bash

```diff
@@ -1,6 +1,11 @@
-cite 'uuid-alias'
-about-alias 'uuidgen aliases'
+# cite 'uuid-alias'
+# about-alias 'uuidgen aliases'
 
-alias uuidu="uuidgen"
-alias uuidl="uuidgen | tr '[:upper:]' '[:lower:]'"
-alias uuid=uuidl # because upper case is like YELLING
+if [ "$(uuid 2>/dev/null)" != "" ]; then # Linux
```

Comment:
Are you testing for the presence of the command? If so, then you can do that using this pattern:

```bash
if _command_exists uuid ; then
```

The `_command_exists` function is provided by Bash-it as a convenience function...

Reply:
Didn't know that.  `kubectl.aliases.bash` didn't use that, so I didn't.  Now, they both do.